bin/magento

```diff
@@ -9,7 +9,7 @@ if [[ -t 1 ]]; then
 fi
 
 exec docker exec "${opts[@]}" \
-	"$(docker-compose ps -q web)" \
+	"$(docker-compose ps -q php8)" \
```

Comment:
I have to find a good way to get the correct container... otherwise I have to pass it all around

Reply:
You could use an environment variable. Or check both and return the one that's running (but you can't guarantee that only one is running). The solution with `container_name` also works, but it loses the namespacing.

Reply:
Yeah... my idea was to use the docker `environment` variable ... unfortunately it creates it only inside the container.